raphtory/src/db/api/mutation/import_ops.rs

```diff
@@ -165,6 +277,63 @@ impl<
         Ok(self.node(node.id()).unwrap())
     }
 
+    fn import_node_as<
+        'a,
+        GHH: GraphViewOps<'a>,
+        GH: GraphViewOps<'a>,
+        V: AsNodeRef + Clone + Debug,
+    >(
+        &self,
+        node: &NodeView<GHH, GH>,
+        new_id: V,
+        force: bool,
+    ) -> Result<NodeView<Self, Self>, GraphError> {
+        if !force {
```

Comment:
If this is all the same code as import_node, I would create an internal function which does the actual work and then have `import_node`/`import_node_as` call it - this way tests cover both and we can't accidentally make them do different things.

We should do the same with edges below